packages/web-app-user-management/src/views/Users.vue

```diff
@@ -484,11 +484,8 @@ export default defineComponent({
          */
         this.selectedUsers = [editUser]
 
-        this.showMessage({
-          title: this.$gettext('User was edited successfully')
-        })
+        bus.publish('app.user-management.entity.saved')
       } catch (error) {
-        console.error(error)
```

Comment:
Why did you delete this line? 🤔 IMO makes sense to log errors to the console.

Reply:
ooops, recovered

Reply:
Thanks :-)